nrf_modem/doc/sockets/socket_options_func.rst

```diff
@@ -340,21 +342,21 @@ NRF_SO_SEC_DTLS_CONN_SAVE
 
    Once the DTLS context is saved, the socket can't be used before the DTLS context is loaded with :c:macro:`NRF_SO_SEC_DTLS_CONN_LOAD`.
 
-   This option fails with nrf_errno ``NRF_EAGAIN`` if an error happened during serialization of the SSL context.
+   This option fails with ``NRF_EAGAIN`` if an error happened during serialization of the SSL context.
```

Comment:
I think we should keep `nrf_errno` or `errno` to clarify that this is not the function return value.
```suggestion
   This option fails with errno ``NRF_EAGAIN`` if an error happened during serialization of the SSL context.
```